ale_linters/ansible/ansible_lint.vim

```diff
@@ -18,28 +18,16 @@ function! ale_linters#ansible#ansible_lint#Handle(buffer, lines) abort
         endif
     endfor
 
-    " Matches patterns line the following:
-    "
-    " test.yml:35: [EANSIBLE0002] Trailing whitespace
-    let l:pattern = '\v^([a-zA-Z]?:?[^:]+):(\d+):?(\d+)?: \[?([[:alnum:]]+)\]? (.*)$'
+    let l:pattern = '\v^([^:]+):(\d+):%((\d+):)? ([-[:alnum:]]+) (.*)$'
     let l:output = []
 
     for l:match in ale#util#GetMatches(a:lines, l:pattern)
-        let l:code = l:match[4]
-
-        if l:code is# 'EANSIBLE0002'
-        \&& !ale#Var(a:buffer, 'warn_about_trailing_whitespace')
-            " Skip warnings for trailing whitespace if the option is off.
-            continue
-        endif
-
         if ale#path#IsBufferPath(a:buffer, l:match[1])
             call add(l:output, {
             \   'lnum': l:match[2] + 0,
             \   'col': l:match[3] + 0,
             \   'text': l:match[5],
-            \   'code': l:code,
-            \   'type': l:code[:0] is# 'E' ? 'E' : 'W',
+            \   'code': l:match[4],
```

Comment:
We should check what the code is and be sure what we're setting. The only supported codes by Vim are `'E', 'W'`, and `'I'`,

Reply:
_ansible-lint_ (version >=5.0.0) can report following severity codes: “VERY_HIGH”, “HIGH”, “MEDIUM”, “LOW”, “VERY_LOW”, “INFO”. Since they can't be mapped 1-to-1 to vim codes, I would prefer just to mark every finding as an `E` and keep it simple, than to develop a biased mapping.

Reply:
I would prefer to differentiate at least between high severity and low severity issues. Specially when dealing with hundreds of issues from roles that I do not maintain but need to use.

What about mapping like this:

VERY_HIGH, HIGH ->  E
MEDIUM, LOW, VERY_LOW -> W
INFO -> I